src/tail/tail.rs

```diff
@@ -346,7 +346,18 @@ fn follow<T: Read>(readers: &mut [BufReader<T>], filenames: &[String], settings:
     let mut read_some = false;
     let mut process = platform::ProcessChecker::new(settings.pid);
 
+    let mut buffer = BufReader::new(stdin());
     loop {
+        loop {
+            let mut datum = String::new();
+            match buffer.read_line(&mut datum) {
+                Ok(0) => break,
+                Ok(_) => {
+                  print!("input consumed: {}", datum);
```

Comment:
all works. travis is passing. appveyor hangs.

https://github.com/albfan/coreutils

I guess I need a timeout on `read_line`. Is there such thing on read_line?

https://stackoverflow.com/questions/42730169/fill-a-string-buffer-from-a-thread